proposals/0069_prometheus-exporters-otel-embedding.md

```diff
@@ -0,0 +1,286 @@
+
+* **Owners:**
+  * [@ArthurSens](https://github.com/ArthurSens)
+
+* **Implementation Status:** Not implemented
+
+* **Related Issues and PRs:**
+  * https://github.com/prometheus/exporter-toolkit/pull/357
+  * https://github.com/prometheus/node_exporter/pull/3459
+  * https://github.com/ArthurSens/prometheus-opentelemetry-collector (proof of concept)
+
+* **Other docs or links:**
+
+> TL;DR: This proposal introduces a mechanism to embed Prometheus exporters as native OpenTelemetry Collector receivers, reducing duplication of effort between the two ecosystems and enabling the "single binary" promise for telemetry collection without forcing reimplementation of hundreds of existing Prometheus exporters.
+
+## Why
+
+The OpenTelemetry Collector ecosystem faces a significant challenge: many components in collector-contrib are "drop-in" replacements for existing Prometheus exporters but often become unmaintained before reaching stability. This duplication of effort occurs because the promise of "one binary to collect all telemetry" is valuable to users, leading to reimplementation of functionality that already exists in mature Prometheus exporters.
+
+This issue became particularly visible during OpenTelemetry's CNCF Graduation attempt, where feedback highlighted that users often feel frustrated when upgrading versions. In response, the Collector SIG decided to be stricter about accepting new components and more proactive in removing unmaintained or low-quality ones.
+
+Meanwhile, the Prometheus community has developed hundreds of exporters over many years, many of which are stable. Creating parallel implementations in the OpenTelemetry ecosystem wastes community resources and often results in "drive-by contributions" that are abandoned shortly after acceptance.
+
+### Pitfalls of the current solution
+
+1. **Duplication of Work**: Infrastructure monitoring receivers are reimplemented in OpenTelemetry when functionally equivalent Prometheus exporters already exist.
+
+2. **Unmaintained Components**: Many OpenTelemetry receivers that replicate Prometheus exporter functionality become unmaintained in early development stages.
+
+3. **Quality and Stability Issues**: The pressure to provide comprehensive coverage leads to accepting components that may not meet quality standards, contributing to collector-contrib's stability problems.
+
+4. **Diverging Ecosystems**: Two communities are solving the same problems independently, fragmenting effort and expertise.
+
+5. **Maintenance Burden**: Both ecosystems must independently maintain similar functionality for monitoring the same infrastructure components.
+
+## Goals
+
+* Enable embedding of Prometheus exporters as native OpenTelemetry Collector receivers via the OpenTelemetry Collector Builder (OCB).
+* Reduce duplication of effort between Prometheus and OpenTelemetry communities.
+* Maintain the "single binary" promise for users who want comprehensive telemetry collection.
```

Comment:
```suggestion
* Maintain the "single binary" promise for users who follow the single binary approach.
```

Users who want comprehensive telemetry collection can also have it with multiple binaries. The need for a single binary really only arises from the wish for a single binary itself.